blog/2022-08-31-cwa-2-26/index.md

```diff
@@ -0,0 +1,18 @@
+---
+page-title: "Project team releases CWA version 2.26 with improvements"
+page-description: "Project team releases CWA version 2.26 with improvements"
+page-name: cwa-2-26
+page-name_de: cwa-2-26
+author: CWA-Team, 10am
+layout: blog
+---
+
+
+The project team of the Robert Koch-Institute (RKI), Deutsche Telekom, and SAP have released **version 2.26 of the Corona-Warn-App (CWA)**. In the new version, the team has made improvements and adjustments, as well as fixed minor bugs. 
```

Comment:
The English name of RKI is spelled without a hyphen. It should be Robert Koch Institute.
See https://www.rki.de/EN/Service/Imprint/imprint_node.html.